pytests/conftest.py

```diff
@@ -17,17 +17,17 @@
 import shutil
 import pytest
 import atexit
+import platform
 import requests
 import subprocess
 import configparser
-import distutils.spawn
```

Comment:
This is a significant change, distutils is deprecated and will be removed from python-3.12